apps/registration-api/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM waggle/beekeeper-key-tools
+WORKDIR /registration-api
+ADD . /registration-api
+# RUN apt-get install curl && curl -sL https://deb.nodesource.com/setup_16.x | bash
```

Comment:
```
RUN apt-get install curl -y && curl -sL https://deb.nodesource.com/setup_16.x | bash
``` 
should work here (for node 16).  The `-y` will accept the prompt so that the curl install doesn't fail.  🎉